client_token.go

```diff
@@ -1,11 +1,21 @@
 package braintree
 
+// ClientTokenRequest represents the parameters for the client token request
 type ClientTokenRequest struct {
-	XMLName    string `xml:"client-token"`
-	CustomerID string `xml:"customerId,omitempty"`
-	Version    int    `xml:"version"`
+	XMLName           string                     `xml:"client-token"`
+	CustomerID        string                     `xml:"customerId,omitempty"`
+	MerchantAccountID string                     `xml:"merchantAccountId,omitempty"`
```

Comment:
Well, I changed it but I cannot guarantee it is the required tag name for the XML since even the official library it is not consistent about that. Please take a look here:

https://github.com/braintree/braintree_java/blob/2def3f6dcf3fbbfcb3c74d67716aadfa3f0658bd/src/main/java/com/braintreegateway/SubscriptionTransactionRequest.java#L36
https://github.com/braintree/braintree_java/blob/2def3f6dcf3fbbfcb3c74d67716aadfa3f0658bd/src/main/java/com/braintreegateway/PaymentMethodRequest.java#L137
https://github.com/braintree/braintree_java/blob/2def3f6dcf3fbbfcb3c74d67716aadfa3f0658bd/src/main/java/com/braintreegateway/ClientTokenOptionsRequest.java#L53

It would be easier if the documentation included the schema (WSDL?) for the XML instead of referring to the SDKs variable names.

Do you have any insight about this?.

Reply:
The different SDKs (Ruby, Java, .NET, etc) all use different casing for the fields and they all seem to be interpreted appropriately. The Java SDK uses camelCase as you pointed out in many places. The Ruby SDK appears to exclusively use kebab case as can be seen in it's [XML generator](https://github.com/braintree/braintree_ruby/blob/2df252641403a86e2212d9c9b3bd8849569c3e5d/lib/braintree/xml/generator.rb#L101). Both camelCase and kebab-case can be used for each field